docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -142,6 +142,15 @@ The progress of files created per shard is also captured in the shard_file_proce
 
 To confirm that the records have indeed been written to the source database, best approach is to check the record count on the source database, if that matches the expected value. Note that verifying data takes more than just record count matching. The suggested tool for the same is [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator).
 
+#### Tracking which shards are lagging
+
+The following sample SQL gives the shards which are yet to catchup in the writer job. The SQL needs to be fired on the metatdata database. Replace the run_id with the relevant run identifier and the window interval with appropriate value.
```

Comment:
typo: metadata*

Do we want to add a link so the user is sure which metadata db we want them to use b/w smt metadata/the one used by changestream/one used by reverse replication job.

Reply:
Done.